docs/.vuepress/theme/components/Footer.vue

```diff
@@ -17,6 +18,29 @@
   </footer>
 </template>
 
+<script>
+  export default {
+    computed: {
+      lastUpdatedDate() {
+        const moment = require('moment')
```

Comment:
Let's move this require statement outside the computed property. Remove this line. Just below the `<script>` tag on line 21, you should be able to add a line: ```import moment from 'moment';```